docs/rule/no-action.md

```diff
@@ -101,6 +101,17 @@ to
 </select>
 ```
 
+
+```hbs
+<myComponent @onValidationChange={{action "onDateValidation"}} />
+```
+
+to 
+
+```hbs
+<myComponent @onValidationChange={{this.onDateValidation}} />
+```
+
```

Comment:
Components are usually capitalized.

```suggestion
```hbs
<MyComponent @onValidationChange={{action "onDateValidation"}} />
```

to 

```hbs
<MyComponent @onValidationChange={{this.onDateValidation}} />
```

```